pkg/kube/inject/testdata/inject/hello.yaml.cni.injected

```diff
@@ -14,11 +14,7 @@ spec:
   template:
     metadata:
       annotations:
-        sidecar.istio.io/interceptionMode: REDIRECT
-        sidecar.istio.io/status: '{"version":"","initContainers":["istio-validation"],"containers":["istio-proxy"],"volumes":["istio-envoy","istio-certs"],"imagePullSecrets":null}'
-        traffic.sidecar.istio.io/excludeInboundPorts: "15020"
```

Comment:
Likewise, those other annotations not being injected is a regression in new installer. This breaks Istio CNI.

Reply:
Its possible this is a bug in my tests not setting things up properly. I am test it out and report back

Reply:
The Helm template has this:
https://github.com/istio/istio/blob/6d0168814728e6a460959035e4b805ef66151a7a/install/kubernetes/helm/istio/files/injection-template.yaml#L436-L445
And that is missing from the new installer's template.